server/api/apps/shared_models/models/quiz_models.py

```diff
@@ -19,19 +19,29 @@ class QuestionType(models.TextChoices):
         settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True
     )
     date_created = models.DateTimeField(auto_now_add=True, null=True)
+    is_verified = models.BooleanField(default=False)
+    mark = models.PositiveIntegerField(null=True)
 
     def __str__(self):
         return self.text
 
 
-class Tag(models.Model):
+class Subject(models.Model):
```

Comment:
Please add some documentation for the new Subject and Topic system and remove the old Tag system documentation! (documentation/docs/backend/quiz/shared_models.md)